README.md

```diff
@@ -700,6 +701,9 @@ const options = {
   // The time in milliseconds that cache data remains fresh.
   cacheLife: 0,
 
+  // Sets wether to persist data ot not after page refresh. Only available on Browser.
```

Comment:
Sets whether to persist the cache after page refresh. Only available in Browser.

Reply:
`// allows caching to persist after page refresh. Only available in Browser.`